src/client/src/tscSQLParser.c

```diff
@@ -1804,9 +1804,8 @@ int32_t validateOneColumn(SSqlCmd* pCmd, TAOS_FIELD* pColField) {
 }
 
 /* is contained in pFieldList or not */
```

Comment:
Check existence of a name against the part of pFieldlist that have already been processed.